### PR TITLE
[TWEAK] Activate and Drop now Extend and Retract arm implants in THE ARM

### PR DESCRIPTION
**Pull request by larentoun**
## What Does This PR Do

Now, when you actiavte you empty hand with an implant, a radial menu opens. When you drop your hand with an implant active, it retracts

Also, it adds an ability to use an empty hand. So, probably, add xeno's spit into the hand by using it? :eyes:

## Why It's Good For The Game

More straight-forward arm implant usage?

## Testing

Implant armtoolset, use it all the ways possible

## Changelog
:cl:
tweak: Activate and Drop now Extend and Retract arm implants in THE ARM. Z/Q (by defaults) Extend and Retracts.
/:cl:

### CODE REVIEW

**Comment by matttheficus** [comment]
I dont really like the idea of misclicking an open hand giving a radial menu to be honest. Dropping already drops the current tool and reopens the radial.

**Comment by S34NW** [comment]
No commits in three weeks, marking as stale.

File: code/modules/surgery/organs/augments_arms.dm [hunk 1]
```diff
@@ -1,58 +1,69 @@
...
+/obj/item/organ/internal/cyberimp/arm/remove(mob/living/carbon/arm_owner, special = 0)
 	Retract()
+	if(hand)
+		UnregisterSignal(hand, COMSIG_ITEM_ATTACK_SELF)
+		UnregisterSignal(arm_owner, COMSIG_MOB_WILLINGLY_DROP)
 	. = ..()
 
+/obj/item/organ/internal/cyberimp/arm/proc/on_item_attack_self()
+	SIGNAL_HANDLER
+	INVOKE_ASYNC(src, .proc/ui_action_click)
```
Comment (SteelSlayer) [inline code/modules/surgery/organs/augments_arms.dm:65]:
```suggestion
	INVOKE_ASYNC(src, PROC_REF(ui_action_click))
```

**Review by SteelSlayer — CHANGES_REQUESTED** [review]

**Comment by larentoun** [comment]
No reason to support upstream if they can't do a little thing to support downstream.

**Pull request closed without merging**